pkg/printers/text.go

```diff
@@ -56,10 +58,18 @@ func (p *Text) Print(ctx context.Context, issues []result.Issue) error {
 }
 
 func (p Text) printIssue(i *result.Issue) {
-	text := p.SprintfColored(color.FgRed, "%s", strings.TrimSpace(i.Text))
+	issueColor := color.FgRed
+	if i.Severity == "warning" {
+		issueColor = color.FgYellow
+	}
+
+	text := p.SprintfColored(issueColor, "%s", strings.TrimSpace(i.Text))
 	if p.printLinterName {
 		text += fmt.Sprintf(" (%s)", i.FromLinter)
 	}
+	if p.printSeverity {
```

Comment:
The only time I really found this useful is if coloring is turned off, so I'm happy to remove it / restructure this.